ansible-tests/validations/check_openstack_endpoints.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: undercloud
+  vars:
+    metadata: Check access to various openstack services
+    description: >
+      This will attempt to connect to each service endpoint defined in your hosts file's controller_vip variable.
+      You must also set the appropriate credentials in validations/auth-header.json
+  tasks:
+  #Check connectivity to horizon
+  - name: Check Horizon
+    uri: url=http://{{controller_vip}} 
+
+  #Check that we can obtain an auth token from horizon
+  - name: Check Keystone
+    uri: 
+     url: http://{{controller_vip}}:5000/v2.0/tokens
+     method: POST
+     HEADER_Content-Type: "application/json"
+     #to-do: come up with a better path mechanism to not hard-code paths
+     body: "{{ ' ' + lookup('file','/home/stack/clapper/ansible-tests/validations/auth-header.json')}}"
```

Comment:
You can put `auth-header.json` in the `ansible-tests/validations/files/` directory and then just call `lookup('file', 'files/auth-header.json')`.

I was wondering whether we could just put the username, password and tenant name as ansible vars, too. We're trying to do make the validations run without any prior setup if possible. That obviously won't work here, but vars are I think easier than editing a bunch of random files.

Plus, when we get called by the tripleo ui, these values can be passed over automatically. But ultimately, it's you guys who are going to benefit from this so what do you think?